srcalc/src/main/java/gov/va/med/srcalc/domain/model/Rule.java

```diff
@@ -88,28 +88,34 @@ void setId(final int id)
         this.fId = id;
     }
     
+    /**
+     * Returns the {@link ValueMatcher}s that form the conditional part of the rule.
+     * Order is important: each matcher may use SpEL variable references to
+     * previously-matched values.
+     */
     @ElementCollection(fetch = FetchType.EAGER) // eager-load due to close association
     // Override strange defaults.
     @CollectionTable(
             name = "rule_value_matcher",
             joinColumns = @JoinColumn(name = "rule_id"))
-    /**
-     * The {@link ValueMatcher}s that form the conditional part of the rule.
-     * Order is important: each matcher may use SpEL variable references to
-     * previously-matched values.
-     */
     public List<ValueMatcher> getMatchers()
     {
         return fMatchers;
     }
     
+    /**
+     * Sets the {@link ValueMatcher}s that form the conditional part of the rule.
```

Comment:
If you used a `@see` or `@link` to the getter, you could omit some of this repeated info.